### PR TITLE
As per COSDev, use singlequotes for strings

### PR DESCRIPTION
**Pull request by sloria**
## Purpose

Ensure all quoting is consistent with COSDev's style guideline:

> Use singlequotes for strings, unless doing so requires lots of escaping.

https://cosdev.readthedocs.io/en/latest/style_guides/python.html

This makes it easier to search for strings while navigating code.

## Changes

- Added flake8-quotes to detect usages of doublequotes
- Used this quick-and-dirty script to replace doublequotes with singlequotes: https://gist.github.com/sloria/11d3721b1b1c578012664b7a83f3b411

### CODE REVIEW

**Comment by brianjgeiger** [comment]
Mad with power.

**Comment by sloria** [comment]
Ha, if I were mad with power, I'd have merged this in myself. I'm graciously waiting for you or Michael to merge this, though. =)

File: website/search/elastic_search.py [hunk 1]
```diff
@@ -67,9 +67,9 @@
     es.cluster.health(wait_for_status='yellow')
 except ConnectionError as e:
     sentry.log_exception()
-    sentry.log_message("The SEARCH_ENGINE setting is set to 'elastic', but there "
-            "was a problem starting the elasticsearch interface. Is "
-            "elasticsearch running?")
+    sentry.log_message('The SEARCH_ENGINE setting is set to "elastic", but there '
```
Comment (brianjgeiger) [inline website/search/elastic_search.py]: 👍

**Comment by brianjgeiger** [comment]
I don't see any problems, although have fun with the merge conflict. You showed remarkable restraint in not replacing the string concatenation with `.format()`. 🎉 🐧 🎉

**Pull request merged**